src/storages/pluggable/index.ts

```diff
@@ -45,12 +54,8 @@ export function PluggableStorage(options: PluggableStorageOptions): IStorageAsyn
     const keys = new KeyBuilderSS(prefix, metadata);
     const wrapper = wrapperAdapter(log, options.wrapper);
 
-    // subscription to Wrapper connect event in order to emit SDK_READY event
-    wrapper.connect().then(() => {
-      if (onReadyCb) onReadyCb();
-    }).catch((e) => {
-      if (onReadyCb) onReadyCb(e || new Error('Error connecting wrapper'));
-    });
+    // emit SDK_READY event on main client
```

Comment:
is this function about the ready event only, or about having the wrapper connect?

Reply:
onReadyCb is for having the ready event. 
ATM:
- InMemory and InLocalStorage only works with `standalone` mode, so they ignore the `onReadyCb` callback.
- Redis and CustomStorage only works with `consumer` or `consumer_partial` mode, so they must use the `onReadyCb` callback to emit the ready event.